aws/ec2mock_test.go

```diff
@@ -6,9 +6,12 @@ import (
 	"github.com/aws/aws-sdk-go/service/ec2/ec2iface"
 )
 
+const DIP_SPLIT_SIZE = 2
```

Comment:
nit: please use camelCase for go variables e.g. `dipSplitSize` since it shouldn't be exported.